### DISCUSSION TITLE
Re-label EntryType name in Matrix fields, Sections, etc

### DISCUSSION THREAD

**Discussion by rungta** [craftcms/cms#15453] [Ideas] [answered]
The list of EntryTypes in a Craft 5 project can rapidly grow to a large number. We start facing a conundrum when naming an EntryType — go for a name that uniquely labels itself amongst all the other global EntryTypes, or go for a name that makes sense in the authoring context.

Eg: A Matrix field that accepts podcast URLs from various services has the following entry types:
- Spotify
- Apple Podcasts
- YouTube
- Vimeo
- RSS Feed
- Other

The “Other” label works best within the Matrix field, but makes little sense within the global EntryTypes list. And as soon as we have a second or third Matrix field with similar catch-all “Other” entry types, labels start getting real messy.

Another good use case for EntryType relabelling is https://github.com/craftcms/cms/discussions/14726#discussioncomment-8985677

Being able to re-label EntryType names will aid developers in keeping the global EntryTypes list organised with a bit more verbosity without sacrificing AX for authors.

**Comment by mmikkel** [comment, marked as answer]
+100%, already requested here: https://github.com/craftcms/cms/discussions/14779 🙂

**Reply by rungta** [reply]
Ah, of course you have! 😄 I was a little surprised when I wasn’t able to find an existing FR here for some reason. Upvoted!
